docs/migration-guide-2.3-2.4.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Migration Guide 2.3-2.4
+---
+
```

Comment:
Note to self: add instructions that the migration script must be run AFTER upgrade.